views/js/qtiCommonRenderer/renderers/interactions/InlineChoiceInteraction.js

```diff
@@ -77,7 +77,17 @@ define([
 
         _setInstructions(interaction);
 
-        $container.on('change', function(){
+        $container.on('change', function(e){
+            //if tts component is loaded and click-to-speak function is activated - we must fix the situation when select2 prevents tts from working
+            //for this a "one-moment" handler of option click is added and removed after event fired
+            if ($(e.currentTarget).parents('.qti-item').hasClass('prevent-click-handler')) {
+                var $selectedIndex = $(e.currentTarget)[0].options.selectedIndex ? $(e.currentTarget)[0].options.selectedIndex : null;
+                $container.find('option').on('click', function(e) {
```

Comment:
Even if you remove the listener just after it should be better to namespace it, to avoid collision with other possible listeners. Or you could use the one-shot listener mode with `.one()`

Reply:
Will update code.